fix_includes.py

```diff
@@ -2373,6 +2374,12 @@ def main(argv):
                     help='Put comments after the #include lines')
   parser.add_option('--nocomments', action='store_false', dest='comments')
 
+  parser.add_option('--update_comments', action='store_true', default=False,
```

Comment:
Just realised that this arg doesn't do anything unless `--comments` is true as well. 
Should I change this code so that `--update_comments` implicitly sets `flags.comments`?

Reply:
I think that makes sense. It gets a little weird if you combine `--nocomments` and `--update_comments`, but either way one of them has to win, and I think it makes sense for `--update_comments` to take precedence.